.github/workflows/semgrep-compare-argo-workflow.yaml

```diff
@@ -0,0 +1,71 @@
+name: trigger-semgrep-comparison-argo-workflow
```

Comment:
please use the same name than the file here, so when it fails it's easier to find the offending workflow

Reply:
Can you add a toplevel comment in this explaining quickly what this workflow is for, and vaguely how it works.